README.rst

```diff
@@ -0,0 +1,49 @@
+===================
+RDO Project Easyfix
+===================
+
+It is a collection of easyfix or low hanging fruits issues for RDO Project.
+
+I am new to RDO Project, How can i contribute?
+==============================================
+
+* You can check the issues and assign to it yourself.
```

Comment:
You can check [Issues](https://github.com/redhat-openstack/easyfix/issues) and assign some to yourself.